imports/ui/annotation.coffee

```diff
@@ -1,5 +1,7 @@
-annotateContent = (content, annotations, options={})->
-  { startingIndex, endingIndex, selectedAnnotationId } = options
+export annotateContent = (content, annotations, options={}) ->
+  { startingIndex, endingIndex, selectedAnnotationId, pointerEvents} = options
+  pointerEvents ?= true
```

Comment:
Adding the selected class and pointer events class could be done in a more versatile way by adding a "classes" property to the annotations (similar to the attributes property) before they passed into the annotateContent function. It will also allow us to keep call-site specific logic out of the function.